bookstore/src/main/webapp/WEB-INF/template/header.jsp

```diff
@@ -0,0 +1,85 @@
+<%@ page contentType="text/html; charset=UTF-8"%>
+<div class="header">
+	<div class="contact-information row">
+		<div class="col-md-4 border-right">
+			<div class="mail-contact">
+				<span class="fa fa-envelope" aria-hidden="true"></span>
+				trancongdanh@gmail.com
+			</div>
+		</div>
+
+		<div class="col-md-4 border-right">
+			<div class="phone-contact">
+				<i class=" fa fa-phone"></i> +84.776.123.456
+			</div>
+			<div class="social-network">
+				<ul>
+					<li><a href="#"><i class="fa fa-facebook"
+							aria-hidden="true" style="color: black"></i></a></li>
+					<li><a href="#"><i class="fa fa-twitter"
+							style="color: black"></i></a></li>
+					<li><a href="#"><i class="fa fa-pinterest"
+							aria-hidden="true" style="color: black"></i></a></li>
+					<li><a href="#"><i class="fa fa-linkedin"
+							style="color: black"></i></a></li>
+				</ul>
+			</div>
+		</div>
+
+		<div class="col-md-2 border1-right">
+			<div class="lang">
+				<button class="btn btn-secondary dropdown-toggle" type="button"
+					id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true"
```

Comment:
@danhtrancong 
1. Remove type="button"
2. Remove id
3. Remove for the order files